From d81f5f313412df572ae3775fbb0a204b0e600cbe Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Tue, 4 Jun 2019 19:52:07 +0200
Subject: Make blink::LayoutUnit::HasFraction constexpr

Other HasFraction methods as in PhysicalUnit are declared already
constexpr and using it. It breaks GCC build.

Bug: 819294.
Change-Id: I0c4bd9bd206d45cf31f7fa815ce8533718a425cb

diff --git a/third_party/blink/renderer/platform/geometry/layout_unit.h b/third_party/blink/renderer/platform/geometry/layout_unit.h
index f073986fbe17..b6dbc761c967 100644
--- a/third_party/blink/renderer/platform/geometry/layout_unit.h
+++ b/third_party/blink/renderer/platform/geometry/layout_unit.h
@@ -202,7 +202,9 @@ class LayoutUnit {
     return value_ > 0 ? LayoutUnit() : *this;
   }
 
-  bool HasFraction() const { return RawValue() % kFixedPointDenominator; }
+  constexpr bool HasFraction() const {
+    return RawValue() % kFixedPointDenominator;
+  }
 
   LayoutUnit Fraction() const {
     // Compute fraction using the mod operator to preserve the sign of the value
-- 
2.20.1

